### ISSUE TITLE
RGB indicator lines under histogram are not disabled when opening editor in raw histogram mode

### ISSUE THREAD

**Issue by heckflosse** [Beep6581/RawTherapee#4829]
to reproduce

1) open raw image
2) set histogram mode to raw histogram
3) close rt
4) start rt
5) open raw image
6) hover over the picture => rgb indicators are shown, wrong
7) set histogram mode to normal (non raw) histogram
8) set histogram mode to raw histogram
9) hover over the picture => rgb indicators are not shown, correct

This fixes it:
```diff
diff --git a/rtgui/histogrampanel.cc b/rtgui/histogrampanel.cc
index e50a4a72a..6ff48a9fd 100644
--- a/rtgui/histogrampanel.cc
+++ b/rtgui/histogrampanel.cc
@@ -443,7 +443,7 @@ bool HistogramRGBArea::getShow()
 
 void HistogramRGBArea::updateBackBuffer (int r, int g, int b, const Glib::ustring &profile, const Glib::ustring &profileW)
 {
-    if (!get_realized () || !showMode) {
+    if (!get_realized () || !showMode || rawMode) {
         return;
     }
 
@@ -606,10 +606,6 @@ void HistogramRGBArea::updateOptions (bool r, bool g, bool b, bool l, bool c, bo
         barDisplayed = false;
     }
 
-    // Disable (but don't hide it) the bar button when RAW histogram is displayed
-    if (rawMode) {
-        showMode = false;
-    }
 }
 
 void HistogramRGBArea::on_realize ()

```

ping @Thanatomanic

**Comment by Thanatomanic** [comment]
Good catch. Fixed with 82187ae.

**Comment by Thanatomanic** [comment]
@da-phil reported a typo of mine. Fixed with 7e36e86
